cmd/provisioner-localpv/app/provisioner_blockdevice.go

```diff
@@ -67,8 +68,10 @@ func (p *Provisioner) ProvisionBlockDevice(opts pvController.VolumeOptions, volu
 	// TODO
 	// VolumeMode will always be specified as Filesystem for host path volume,
 	// and the value passed in from the PVC spec will be ignored.
-	fs := v1.PersistentVolumeFilesystem
-
+	volumeMode := v1.PersistentVolumeFilesystem
+	if *opts.PVC.Spec.VolumeMode == v1.PersistentVolumeBlock {
+		volumeMode = v1.PersistentVolumeBlock
```

Comment:
set the path to blkPath here.

Reply:
n_: can this if condition be avoided. refactor the code by moving the WithVolumeMode in the if condition below.